climada/util/coordinates.py

```diff
@@ -459,6 +460,136 @@ def get_gridcellarea(lat, resolution=0.5, unit="ha"):
     return area
 
 
+def compute_grid_cell_area_validation(res: float) -> tuple[np.ndarray]:
```

Comment:
I am confused by the name of this function. It says that it computes the area of grid cells. So why is called `validation`?